src/main/resources/testchipip/bootrom/bootrom.S

```diff
@@ -1,4 +1,4 @@
-#define BOOTADDR_REG 0x4000
+#define BOOTADDR_REG 0x1000
```

Comment:
To be clear, this isn't related/required for the bootrom address changes right?

Reply:
Yes, this is not related to the bootrom address change.

This change is corresponding to the proposal mentioned [here](https://docs.google.com/presentation/d/18qrFsHImYO4OJEpl8oQ_g3m2dc6a2Q0LppnVxTX-19I/edit#slide=id.g1c27f282ad8_0_54).